android/capacitor/src/main/java/com/getcapacitor/CapConfig.java

```diff
@@ -436,6 +449,7 @@ public static class Builder {
         private Boolean webContentsDebuggingEnabled = null;
         private boolean loggingEnabled = true;
         private boolean initialFocus = false;
+        private int minWebViewVersion = 60;
```

Comment:
```suggestion
        private int minWebViewVersion = DEFAULT_ANDROID_WEBVIEW_VERSION;
```
use the `DEFAULT_ANDROID_WEBVIEW_VERSION` constant instead of hardcoding the value in two places